contrib/ansible/deploy-devel.yaml

```diff
@@ -5,6 +5,9 @@
   gather_facts: no
   vars:
     aws_section: default
+    use_real_aws: false
```

Comment:
Lets document the use of this in the README.

Reply:
Yep. Good catch.